core/src/avm2/globals/flash/events/ThrottleType.as

```diff
@@ -1,6 +1,7 @@
 package flash.events
 {
 
+    [API("676")]
```

Comment:
I think this should be `674` for Flash Player 11:
https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/events/ThrottleType.html